setup.py

```diff
@@ -18,12 +18,19 @@ def load_version() -> str:
         return re.search(r'^__version__ = [\'"]([^\'"]*)[\'"]', f.read(), re.M).group(1)
 
 
+NLP_REQUIRE = load_requirements("ci/requirements_nlp.txt")
+
+
 setup(
     # technical things
     version=load_version(),
     packages=find_packages(exclude=["ci", "docs", "pipelines", "tests*"]),
     python_requires=">=3.8,<4.0",
     install_requires=load_requirements("ci/requirements.txt"),
+    extras_require={
+        "nlp": NLP_REQUIRE,
```

Comment:
@AlekseySh 
Let's discuss here in some "thread"

>what if I installed library without nlp? I will get error when imports chain will be executed

Why do you need to import nlp related parts without nlp related dependencies? 

Or more general question: will `Any` typing also be used in the CV and audio domains for transforms and models?

Reply:
I think, proper solution - import directly from dependencies and if dependencies are not installed - show message to install them or install oml with extra (oml[nlp], oml[cv], oml[audio]).

We can add some helpers and call them before any imports. Like 

```
verify_cv_installed()
import os

import cv2
```

Reply:
I will do it on next step - separate cv dep's.

Reply:
> Why do you need to import nlp related parts without nlp related dependencies?

because we have some import in `__init__.py`, that is why an error may be triggered 

generally you are right, we can rework it in the next PRs

Reply:
Sure. CV is just an example to "think about". My whole point is not about `nlp` itself, but about any domain

Reply:
this PR is about requirement files and pip tags, let's keep it simple and rework python code in the next ones